src/v2/Apps/_Preferences2/PreferencesApp.tsx

```diff
@@ -1,9 +1,176 @@
-import { Text } from "@artsy/palette"
+import {
+  Box,
+  Button,
+  Checkbox,
+  Column,
+  Flex,
+  GridColumns,
+  Separator,
+  Spacer,
+  Text,
+} from "@artsy/palette"
+import { useState } from "react"
 
 export const PreferencesApp: React.FC = () => {
+  const [uncheckedAll, setUncheckedAll] = useState(false)
+
   return (
     <>
-      <Text variant="xl">Preferences App</Text>
+      <Text variant="xl" mt={4} mb={6}>
+        Preferences Center
+      </Text>
+
+      <Separator mt={2} />
+
+      <GridColumns>
+        <Column
+          span={12}
```

Comment:
So all this is doing is just creating one big column. What you want is two columns right? Like the first column contains the description and maybe spans 8. The second column spans the remaining, 4, and contains the checkbox.

Reply:
Updated.